<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignatureDialog</class>
 <widget class="QWidget" name="SignatureDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="SignatureDialogLayout">
   <item>
    <widget class="QLabel" name="title"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
	 <widget class="QWidget" name="errorTab">
      <attribute name="title">
       <string>Error</string>
      </attribute>
	   <layout class="QVBoxLayout" name="errorLayout">
        <item>
         <widget class="QLabel" name="info">
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="more">
          <property name="text">
           <string>&lt;a href=&quot;info&quot;&gt;More info&lt;/a&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="error">
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="errorScrollSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
     </widget>
     <widget class="QWidget" name="signer">
      <attribute name="title">
       <string>Signer</string>
      </attribute>
      <layout class="QHBoxLayout" name="signerLayout">
       <item>
        <widget class="QGroupBox" name="signerLocationGroup">
         <property name="title">
          <string>Signer location</string>
         </property>
         <layout class="QFormLayout" name="signerLocationGroupLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="signerCityLabel">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="signerCity">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="signerStateLabel">
            <property name="text">
             <string>State</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="signerState">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="signerCountryLabel">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="signerCountry">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="signerZipLabel">
            <property name="text">
             <string>Zip</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="signerZip">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="signerRoleGroup">
         <property name="title">
          <string>Role / resolution</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>165</width>
               <height>162</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="signerRoleGroupLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeWidget" name="signatureView">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Signature</string>
      </attribute>
      <column>
       <property name="text">
        <string>Attribute</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SignatureDialog</receiver>
   <slot>buttonClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClicked(QAbstractButton*)</slot>
 </slots>
</ui>
